extension/docs/HowToRun.md

```diff
@@ -18,21 +18,61 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Environment variables
+## Environment variable
+
+Extension module requires 1 environment variable to start. This environment variable name is `ADYEN_INTEGRATION_CONFIG` and it must contain settings as attributes in a JSON structure.
```

Comment:
This is not a typical environment variable, maybe we should give a hint that could be added by removing all spaces on the JSON and making it one line, sth like: `JSON.stringify(config)` :) otherwise, it will be hard to export a multiline environment variable? Or is there an easy way? Do you want to give an export example in docs?

Reply:
I don't think it matters if it's one line or not. I tried it with multiple lines and it still works.

Reply:
You probably using `"` before and after JSON, my first experience sth like, also not considering invalid keys in API keys because of quotes, let me know how you are exploring it to an env variable :)
 
<img width="491" alt="Screenshot 2021-02-05 at 14 15 59" src="https://user-images.githubusercontent.com/3469524/107038382-b6018e80-67bc-11eb-91a0-1bb3a9ed0bb8.png">

Reply:
Yes, I used the single quotes :)